sspl_test/framework/utils/encryptor.py

```diff
@@ -37,5 +37,5 @@ def encrypt(key, text):
 
 def decrypt(key, text, caller=None):
     """Decrypt the <text>."""
-    decrypt_text = Cipher.decrypt(key, text).decode()
+    decrypt_text = Cipher.decrypt(key, text.encode("utf-8")).decode("utf-8")
```

Comment:
Why do we encode/decode? Just passing the `text` won't work?

Reply:
We are not sure about the type how this can be stored and returned back from conf_store. Its upto provisioner.
We are following the guidance from [provisioner doc](https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/127893714/Cipher+key+generation+for+secret+password)